roles/cinder-common/tasks/main.yml

```diff
@@ -35,7 +35,13 @@
     - /var/lib/cinder/lock
 
 - name: cinder log dir
-  file: dest=/var/log/cinder state=directory mode=0755 owner=cinder group=cinder
+  file: dest=/var/log/cinder state=directory mode=2750 owner=cinder group=adm
+
+- acl: name=/var/log/cinder state=present entry={{ item }}
```

Comment:
`entry` argument is deprecated. The preferred method is to use 
`entity`, `etype`, and `permissions`. You'll want to construct your loop differently, to loop over a hash for this data.